support-frontend/assets/helpers/tracking/acquisitions.js

```diff
@@ -59,6 +60,7 @@ export type PaymentAPIAcquisitionData = {|
   source: ?string,
   abTests: ?AcquisitionABTest[],
   gaId: ?string,
+  transactionFee?: number,
```

Comment:
👍

Reply:
we'll need to update the model in payment-api before putting this live, but this is the right approach.